synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -233,6 +233,31 @@ void Dock_Timetrack2::setup_tool_palette()
 		action_button_map[tool_button->get_name()] = tool_button;
 		tool_item_group->add(*tool_button);
 	}
+
+	struct interpolationButtonInfo {
+		std::string icon_name;
+		std::string name;
+		synfig::Interpolation interpolation;
+	};
+
+	const std::vector<interpolationButtonInfo> interp_buttons_info{
+		{"interpolation_type_clamped_icon", N_("Clamped"), synfig::INTERPOLATION_CLAMPED},

```

Comment:
Please rebase and use `interpolation_icon_name` from `iconcontroller.cpp` to get icon names.